{{cookiecutter.project_slug}}/production.yml

```diff
@@ -51,11 +51,16 @@ services:
     <<: *django
     image: {{ cookiecutter.project_slug }}_production_celeryworker
     command: /start-celeryworker
+    env_file:
+      - ./.envs/.production/.django
+      - ./.envs/.production/.postgres
```

Comment:
They should already be included in [the `django` anchor](https://yaml.org/spec/1.2/spec.html#id2785586) we define at the top:

https://github.com/pydanny/cookiecutter-django/blob/2698867cf8b949b03ed215883f51538fe2f04b38/%7B%7Bcookiecutter.project_slug%7D%7D/production.yml#L9-L19

Note the `django: &django` at the top, which expanded at the line 51 and 59 with `<<: *django`